README.md

```diff
@@ -67,6 +101,14 @@ p result.data      # => {}
 
 Push Metric
 
+If you use user token + `org_name/bucket_name`:
+```ruby
+result = zeus_client.bucket("your_org_name/your_bucket_name").send_metrics([{point: {value: 1, ...}}, ...])
```

Comment:
Same expression are continuing. Can you compress these descriptions? (Or make wiki in this repo for detail)